scripts/set-version.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env python3
```

Comment:
Because pack no longer sets the version we need the version information filled in for the core SDKs so that when we call pack on them from the conformance tester they work.

This is _one_ approach to solve that problem. Just make sure we set sdk/.version and sdk/nodejs/package.json and sdk/python/lib/setup.py with the same version all at once.

An _alternative_ is to special case the core library inside the "pack" command of each language to assume we're running in pu/pu for conformance testing if the library is "pulumi" and find the .version and use it to fill in the version before packing using the envvar and copy tricks from before.

Reply:
Note there's some more bits to do on this e.g. the _version.py file ought to be updated as well for python.